lib/litle/api.rb

```diff
@@ -4,9 +4,10 @@ class PaymentPlugin < ::Killbill::Plugin::ActiveMerchant::PaymentPlugin
 
       def initialize
         gateway_builder = Proc.new do |config|
-          ::ActiveMerchant::Billing::LitleGateway.new :login => config[:login],
+          ::ActiveMerchant::Billing::LitleGateway.new :login => config[:username],
```

Comment:
Because the config file uses that key:
https://github.com/killbill/killbill-litle-plugin/blob/master/litle.yml#L4